src/lib/relay/QueryRenderers.tsx

```diff
@@ -280,24 +281,28 @@ interface CollectionRendererProps extends RendererProps {
 }
 
 export const CollectionRenderer: React.SFC<CollectionRendererProps> = ({ collectionID, render }) => (
-  <QueryRenderer<QueryRenderersCollectionQuery>
-    environment={environment}
-    query={graphql`
-      query QueryRenderersCollectionQuery($collectionID: String!) {
-        collection: marketingCollection(slug: $collectionID) {
-          ...Collection_collection
+  console.log(Dimensions.get("screen").width, "SCREEEEEEEEN WIDTH"),
```

Comment:
Rogue `console.log`. (I'd use the suggestion feature, but it is multiple lines and I don't think the suggestion feature supports that)

Reply:
Good catch!